ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3622,3 +3622,44 @@ def test_numpy___invert__(
         method_flags=method_flags,
         on_device=on_device,
     )
+    
+# var
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="var",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
You should generate values for all the parameters of the function, `dtype` `ddof`, `keepdims` and `where` are missing here.

Reply:
keepdims=st.booleans(),
where=np_frontend_helpers.where(),
dtype=x=helpers.get_dtypes(kind="integer")
ddof=0

Is it right?